src/data/chaos.ts

```diff
@@ -381,9 +381,9 @@ export const blessings: ChaosBlessing[] = [
     },
   },
   {
-    // ChaosManaBlessing
+    // ChaosMagickBlessing
```

Comment:
While the game calls this resource "magick" in user-facing tooltips, ChaosManaBlessing is the actual name of the object in source code. So for these commented object names, I'd like to make an exception (for future code spelunking or scripting).